examples/backuppolicy.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: backups.anynines.com/v1beta3
+kind: BackupPolicy
+metadata:
+  name: backuppolicy-sample
+spec:
+  enabled: true
+  scheduleConfig:
+    type: "Cron"
+    cron:
+      expression: "*/5 * * * *"
+  serviceInstance:
+    name: "sample-pg-cluster"
+    kind: "Postgresql"
```

Comment:
`Postgresql` will fail as the a8s-backup-controller included in a8s-deployment expects `PostgreSQL`.